R/component-LSM.R

```diff
@@ -0,0 +1,292 @@
+# QUESTION: latent.space.pos.m is never used, should we remove it?
```

Comment:
oh, geez! We should keep it, but update the places where the mean for the prior is defaulted to 0.  I added notes in the two main places I saw it being used.